pkg/ddevapp/ddevapp.go

```diff
@@ -3043,7 +3043,12 @@ func getBackupCommand(app *DdevApp, targetFile string) string {
 	case app.Database.Type == nodeps.MariaDB && nodeps.ArrayContainsString(oldMariaVersions, app.Database.Version):
 		fallthrough
 	case app.Database.Type == nodeps.MySQL:
-		c = fmt.Sprintf(`xtrabackup --backup --stream=xbstream --user=root --password=root --socket=/var/tmp/mysql.sock  2>/tmp/snapshot_%s.log | gzip > "%s"`, path.Base(targetFile), targetFile)
+		// This old Debian 9 container has zstd 1.1.2 which doesn't support multithreading.
+		if app.Database.Version == "5.5" || app.Database.Version == "5.6" {
```

Comment:
@stasadev I think you suggested replacing these with `command -v`, but that requires a running container, correct? I was thinking it was better to just rely on the data already available in the function, but if it's OK to do runtime checks I can switch these over.